Fix a bug whereby 'node-guile-version' would pass a node instead of
a session to 'rexec'.

diff --git a/modules/ssh/dist/node.scm b/modules/ssh/dist/node.scm
index 9c065c7..29a3906 100644
--- a/modules/ssh/dist/node.scm
+++ b/modules/ssh/dist/node.scm
@@ -411,7 +411,8 @@ procedure returns the 1st evaluated value if multiple values were returned."
   "Get Guile version installed on a NODE, return the version string.  Return
 #f if Guile is not installed."
   (receive (result rc)
-      (rexec node "which guile > /dev/null && guile --version")
+      (rexec (node-session node)
+             "which guile > /dev/null && guile --version")
     (and (zero? rc)
          (car result))))
